services/src/main/java/org/exoplatform/news/service/impl/NewsServiceImplV2.java

```diff
@@ -470,12 +493,12 @@ private News createDraftArticleForNewPage(News draftArticle, String pageOwnerId,
   }
 
   private Page getArticlesRootPage(String ownerId) {
```

Comment:
not needed IMO, you can use getNoteOfNoteBookByName directly in the caller method